### ISSUE TITLE
[Bug] PHPCS fails with phpcs.xml from repository.

### ISSUE THREAD

**Issue by leewillis77** [php-censor/php-censor#287]
## Expected behavior

PHPCS run using phpcs.xml from repository.

## Actual behavior

PHPCS fails with the following error:
```
RUNNING PLUGIN: PHP Code Sniffer (Stage: Test)
Directory: <BUILD_PATH>/app/
Plugin options: {"standard":"/phpcs.xml","directory":"app","allowed_warnings":0,"allowed_errors":0}
Looking for binary: phpcs, priority = local
Found in <BUILD_PATH>/vendor/bin (local): phpcs
Args: ["cd \"%s\" && <BUILD_PATH>/vendor/bin/phpcs --report=json %s %s %s %s %s \"%s\" %s %s %s","<BUILD_PATH>/"," --standard=<BUILD_PATH>/app/phpcs.xml"," --extensions=php"," --ignore=\"vendor\"","","","<BUILD_PATH>/app/","","",""]
Command: cd "<BUILD_PATH>/" && <BUILD_PATH>/vendor/bin/phpcs --report=json  --standard=<BUILD_PATH>/app/phpcs.xml  --extensions=php  --ignore="vendor"   "<BUILD_PATH>/app/"   
ERROR: the "<BUILD_PATH>/app/phpcs.xml" coding standard is not installed. The installed coding standards are PEAR, MySource, PSR12, Zend, PSR2, PSR1 and Squiz
Execution status: 3
ERROR: the "<BUILD_PATH>/app/phpcs.xml" coding standard is not installed. The installed coding standards are PEAR, MySource, PSR12, Zend, PSR2, PSR1 and Squiz
Exception: Could not process the report generated by PHP Code Sniffer.
PLUGIN: FAILED
```

## Steps to reproduce

I'm running my project with the following in the `.php-censor.yml` for PHPCS. This worked (with 'path' instead of 'directory') on 0.23 consistently. Under 1.0.5 however it fails with the above error.

```yml
php_code_sniffer:
        standard: "/phpcs.xml"
        directory: "app"
        allowed_warnings: 0
        allowed_errors: 0
```

## Environment

* PHP Censor version : 1.0.5
* Operating System : Ubuntu 16.04.6 LTS (Xenial Xerus)
* PHP version : PHP 7.2.15-1+ubuntu16.04.1+deb.sury.org+1
* MySQL/PostgreSQL version: Server version: 5.7.25-0ubuntu0.16.04.2

**Comment by corpsee** [comment]
@leewillis77 Thank you for the bug report!

**Comment by corpsee** [comment]
@leewillis77 You are right, PHPCS plugin working now strange and inconsister with other types of paths and plugins. I will fix behavior and documentation soon.

For now I have workaround for the issue: you may use relative path like: `standard: "../phpcs.xml"` (relative from `BUILD_PATH`).

**Comment by corpsee** [comment]
@leewillis77 Fixed in `release-1.0` branch. Fix will be released with version `1.0.7` soon.

Now you should use relative from `BUILD_PATH` path or absolute path (As other paths like `directory` etc.):
```yml
php_code_sniffer:
        standard: "./phpcs.xml"
        #standard: "phpcs.xml"
        #standard: "%BUILD_PATH%/phpcs.xml"
        #standard: "/home/user/phpcs.xml"
        directory: "app"
        allowed_warnings: 0
        allowed_errors: 0
```

**Comment by leewillis77** [comment]
Awesome - thanks!
